pyne/xs/cache.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, group_struct=None,
                 self.data_sources.append(ds)
         self._cache['E_g'] = _valid_group_struct(group_struct)
         self._cache['phi_g'] = None
+        self._xs_scalers = {}
```

Comment:
Users should be able to pass in a value via the constructor. Namely, `scalars=None` and then this line will be:

```
self._scalars = {} if scalars is None else scalars
```